frontend/providers/adminer/src/interfaces/kubeblock.ts

```diff
@@ -77,11 +77,46 @@ export async function generateKubeBlockClusterTemplate(
 
           const username = atob(secret.body.data['username']);
           const password = atob(secret.body.data['postgres-password']);
-          const host = atob(secret.body.data['host']);
-          const port = atob(secret.body.data['port']);
+          const endpoint = atob(secret.body.data['endpoint']);
 
-          const pgConnection = 'pgsql://' + username + ':' + password + '@' + host + ':' + port;
-          connections.push(pgConnection);
+          const connection = 'pgsql://' + username + ':' + password + '@' + endpoint;
+          connections.push(connection);
+        } else if (clusterType === 'apecloud-mysql') {
```

Comment:
Is it better to use "Map" instead of "If".

```ts
const clusterMap = {
   pg:...
   mysql: ...
}
const password = atob(secret.body.data[clusterMap[clusterType]]);

```